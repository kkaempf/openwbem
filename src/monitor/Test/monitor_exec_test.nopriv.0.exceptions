InsufficientPrivilegesException   CheckException: readDirectory: insufficient privileges
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
