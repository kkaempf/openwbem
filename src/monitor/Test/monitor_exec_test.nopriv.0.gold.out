argv:
  montest;
  arg1;
  arg2;
environ:
  XPATH=XXX;
  IFS= ;
  XPATH=XXX;
  XPATH=XXX;
  OW_PRIVILEGE_MONITOR_DESCRIPTOR=XXX;
  PATH=/bin;
  XPATH=XXX;
  foo=bar;
User: owprovdr

readDirectory
Exception:
  type: PrivilegeManagerException
  msg:  CheckException: readDirectory: insufficient privileges

readDirectory
Exception:
  type: IPCIOException

readDirectory
Exception:
  type: IPCIOException

readDirectory
Exception:
  type: IPCIOException

rename
Exception:
  type: IPCIOException

rename
Exception:
  type: IPCIOException

rename
Exception:
  type: IPCIOException

rename
Exception:
  type: IPCIOException

unlink
Exception:
  type: IPCIOException

unlink
Exception:
  type: IPCIOException

unlink
Exception:
  type: IPCIOException

open
Exception:
  type: IPCIOException

open
Exception:
  type: IPCIOException

open
Exception:
  type: IPCIOException

open
Exception:
  type: IPCIOException

open
Exception:
  type: IPCIOException

open
Exception:
  type: IPCIOException

open
Exception:
  type: IPCIOException

open
Exception:
  type: IPCIOException

open
Exception:
  type: IPCIOException

open
Exception:
  type: IPCIOException

open
Exception:
  type: IPCIOException

open
Exception:
  type: IPCIOException
ERROR OUTPUT: err output
